client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/problem/rpc/RPCExceptionInfoDto.java

```diff
@@ -26,4 +26,9 @@
      * a dto of the exception defined by the RPC service
      */
     public ParamDto exceptionDto;
+
+    /**
+     * if the exception is a cause of java.lang.UndeclaredThrowableException
+     */
```

Comment:
why is this special?

should then rather provide directly the cause name?

Reply:
we handled the detailed info about the cause with this DTO, 
and this property is used to indicate if the exception is UndeclaredThrowableException and the DTO is for its cause.

the doc is updated.